cairo/kakarot-ssj/crates/contracts/Scarb.toml

```diff
@@ -8,16 +8,16 @@ edition = "2024_07"
 [dependencies]
 starknet.workspace = true
 evm = { path = "../evm" }
-openzeppelin = { path = "../openzeppelin" }
 utils = { path = "../utils" }
+openzeppelin = { git = "https://github.com/openzeppelin/cairo-contracts", tag="v0.16.0" }
```

Comment:
last version is 0.18.0 - is it possible to use it?

Reply:
No, it requires scarb version 2.8.4.

Reply:
ok for me to update scarb to 2.8.4 as well

Reply:
Alright, I'll bump scarb and OZ's version to latest